### ISSUE TITLE
Documentation page crashes Chrome when highlighting text

### ISSUE THREAD

**Issue by russellweed** [turbobutton/tbutt-vr#12]
**Describe the bug**
In Chrome, the browser may crash when highlighting sample code on the page. This is done when highlighting code from right-to-left and is reproducible 100% of the time.

I have narrowed this down to the code::before selector in main.css. The exact problem is with the `content: "\00a0";` style. Commenting this out in Chrome's inspector stops the browser from crashing.

**To Reproduce**
1. Visit https://tbutt.net/help/camera/
2. Scroll down to the **Nested Objects** section
3. Try to highlight the "TBCameraRig.instance.GetCenter()" text **going from right to left**
4. The browser will crash when you mouseover the letter T

You may be asking yourself why I'm highlighting from right to left. The ::before CSS declaration mentioned above for some reason makes it so that this text cannot be highlighted when going left to right.

**Expected behavior**
Expected to see the text be highlighted and the browser remain crash free.

**Additional information:**
 - Chrome 76.0.3809.100 (current at the time of this writing)

**Additional context**
If you are unable to reproduce this, please feel free to contact me for further assistance.

**Comment by hlink** [comment]
I can't reproduce this in Firefox or Chrome 76.0.3809.100 on Windows 10. Both also allow me to highlight from left to right as normal.

What OS are you using? I can take a look at the CSS tweak you suggested, but would like to know if there are other things that need to be addressed too.

**Comment by russellweed** [comment]
Windows 10 Pro, 1809.

I also tried this on my work computer and had the same problem.

**Comment by hlink** [comment]
Does it happen in an incognito window (without extensions enabled)?

**Comment by russellweed** [comment]
Yes. I also tried disabling all my extensions (I only have 2 anyway) and it still crashes, Incognito or normal.

**Comment by hlink** [comment]
Thanks for the troubleshooting! I'm going to check with the dev that provides the theme used by our Github Pages site, to see if this is something that can be fixed in the main branch.
